src/test/shell/bazel/toolchain_test.sh

```diff
@@ -1076,6 +1076,117 @@ function test_local_config_platform() {
   expect_log '@local_config_platform//:host'
 }
 
+# Test cycles in registered toolchain dtaa, which can only happen when
```

Comment:
> "dtaa"

not sure if type ("data"?) or bad/copy paste ("... registered_toolchains").

Reply:
Fixed the comment, thanks for catching that.